### PR TITLE
Email upon table upload

### PR DESCRIPTION
**Pull request by gbova**
QA is testing large lookup table uploads for INDDEX, and they wanted a way to record the upload's start and end times so testers don't have to babysit the status page

Relevant QA tickets:
https://dimagi-dev.atlassian.net/browse/QA-703
https://dimagi-dev.atlassian.net/browse/QA-652

### CODE REVIEW

File: corehq/apps/fixtures/tasks.py [hunk 1]
```diff
@@ -1,29 +1,57 @@
+import datetime
 
 from celery.task import task
 
 from dimagi.utils.chunked import chunked
 from soil import DownloadBase
 
+from django.conf import settings
```
Comment (emord) [inline corehq/apps/fixtures/tasks.py]: nit: Our [`isort` config](https://github.com/dimagi/commcare-hq/blob/12925a7b596ca2cf11d2b90bd3fc1f410780ae37/.isort.cfg) specifies:

* core stdlib
* django
* thirdparty
* dimagi libs
* HQ

One day stickler will get flake8-isort

File: corehq/apps/fixtures/views.py [hunk 2]
```diff
@@ -570,6 +571,8 @@ def _get_fixture_upload_args_from_request(request, domain):
             replace = True
         elif replace.lower() == "false":
             replace = False
+        user_email = request.couch_user.email if request.couch_user.email is not None \
+            else request.couch_user.username
```
Comment (emord) [inline corehq/apps/fixtures/views.py]: Is it intentional that all users are going to get this email? Should this be controlled by a feature flag or only go to dimagi emails? Not sure how many people use this API,

Comment (orangejenny) [reply corehq/apps/fixtures/views.py]: How about the SUPPORT feature flag? Not all of QA have dimagi emails or are superusers.

Comment (gbova) [reply corehq/apps/fixtures/views.py]: I liked the idea of putting this behind the `SUPPORT` feature flag, and I've updated this PR to reflect that

**Review by emord — APPROVED** [review]

**Pull request merged**